### DISCUSSION TITLE
How to enable kerberos related authentication debug logs

### DISCUSSION THREAD

**Discussion by breeze7086** [edenhill/librdkafka#4880] [Q&A] [answered]
Hi guys,

How do I enable debug logs when I use librdkafka to do SASL_SSL authentication via Kerberos?
I have tired with enable "debug: all", but it looks like I can't get all logs I want compare with when I use Java client.
In Java client, if I need kerberos authentication debug logs, I could use
```
export KAFKA_OPTS="-Djava.security.auth.login.config=jaas.conf -Dsun.security.krb5.debug=true -Dsun.security.jgss.debug=true -Djavax.net.debug=all"
```

My question is, how to represent these debug configurations by using librdkafka?

PS:
librdkafka version (release number or git tag): `v2.6.0`
 Apache Kafka version: `3.4.1`

**Comment by breeze7086** [comment]
Does the configuration of `log.thread.name` make sense?

**Comment by d6blju** [comment, marked as answer]
Set export KRB5_TRACE=/dev/stderr in the terminal

**Comment by breeze7086** [comment]
Thank you @d6blju 
After I give the environment value of KRB5_TRACE to the program, I can see there has more logs been generated, but it looks like it's still not so complete as Java client Kerberos debug logs. :(

**Comment by d6blju** [comment]
@breeze7086 Try to describe the problem you encountered. You can refer to https://web.mit.edu/kerberos/krb5-1.18/doc/user/user_config/kerberos.html. Note that different krb5 versions will have different configurations.

**Comment by breeze7086** [comment]
It looks like only KRB5_TRACE related to my question. :(
I feel like it's hard to find comprehensive debug logs as Java client has.

**Comment by breeze7086** [comment]
KRB5_TRACE  could be the only way to see debug logs of rdlibkafka
